tests/mdev_configuration_allocation_test.go

```diff
@@ -26,11 +30,45 @@ import (
 var _ = Describe("[Serial][sig-compute]MediatedDevices", func() {
 	var err error
 	var virtClient kubecli.KubevirtClient
+	var nodeLabels = make(map[*k8sv1.Node]map[string]string)
```

Comment:
To make sure nodeLabels is being renewed before every test it's better never initialize variables here, but instead only to declare them. Something like:

```go
var nodeLabels map[*k8sv1.Node]map[string]string
...
BeforeEach(func() {
    nodeLabels = make(map[*k8sv1.Node]map[string]string)
})

Reply:
Also, I think it's safer to have the node's name as a key for the map instead of a pointer. Meaning going with `map[string]map[string]string`

Reply:
Also, nit:
we can also declare a new type to make it a bit more readable, something like
```go
type NodeNameToLabels map[string]map[string]string
```

But maybe it's an overkill :)